src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
It looks like we will have to switch to `"incognito": "spanning"`  mode if we want to do this. FF will soon reject extensions which use  split mode.

https://bugzilla.mozilla.org/show_bug.cgi?id=1277686